docs/src/swagger-output.json

```diff
@@ -3093,48 +3507,65 @@
         }
       }
     },
-    "/companies/{id}": {
+    "/teams/{id}": {
       "get": {
         "tags": ["users"],
-        "summary": "Finds and displays company by id.",
+        "summary": "Finds and displays team by team ID.",
         "description": "",
         "produces": ["application/json"],
         "parameters": [
           {
             "name": "id",
             "in": "path",
             "type": "integer",
-            "description": "ID",
+            "description": "Team ID",
             "required": true
           }
         ],
         "responses": {
           "200": {
             "description": "successful operation",
             "schema": {
-              "type": "array",
-              "items": {
-                "$ref": "#/definitions/Company"
-              }
+              "type": "object",
+              "allOf": [
+                {
+                  "$ref": "#/definitions/Team"
+                },
+                {
+                  "$ref": "#/definitions/Permission"
+                }
+              ]
```

Comment:
you should only be referencing Team here, not Permission as well if you look at the route we havae